packages/front-end/components/Features/RuleModal/index.tsx

```diff
@@ -563,7 +569,7 @@ export default function RuleModal({
       const res = await apiCall<{ version: number }>(
         `/feature/${feature.id}/${version}/rule`,
         {
-          method: i === rules.length ? "POST" : "PUT",
+          method: duplicate ? "POST" : i === rules.length ? "POST" : "PUT",
```

Comment:
When duplicating, we want to set the rule's id in the body to an empty string.  The POST endpoint will only generate an id if it's empty.

Reply:
Good catch